tests.edn

```diff
@@ -0,0 +1,4 @@
+#kaocha/v1
```

Comment:
Since `kaocha` doesn't yet respect `test-ns-hook`, the tests in `selvage.test.flow-test` don't pass. That said, `kaocha` seems able to run test files with `defflow`s that don't depend on auxilary `deftest`s
```clojure
(defflow basic-flow
  (fn [w] (assoc w :basic 1))
  (testing (is (= (:foo *world*) nil)))
  (testing (is (= (:basic *world*) 1))))
```

Reply:
Good to know that someone is actually using `test-ns-hook`. I don't think it would be too hard to implement, the main downside with it is that it's opaque, so you lose a lot of detail in your test report about individual vars.

Reply:
Yeah, I'm not sure what the best way to attack this problem is, so I used `test-ns-hook`.

The problem being defining but not running aux `deftest`s that only make sense when called inside of a `defflow`.

Like from the example in the PR desc:
```clojure
(deftest first-val-0-test
  (testing ":first-val is 0"
    (is (= 0 (:first-val *world*)))))
```

Reply:
Can you change the `deftest` to a `defn`?

Reply:
Indeed! As long as I add some meta data to the `defn` (I'll define a `defcheck` macro or something). Thanks, this is much cleaner than using `test-ns-hook`!